client/locales/en-US.json

```diff
@@ -31,6 +31,13 @@
   "darkModeToggle": "Toggle dark mode",
   "dateAdded": "Date added",
   "details": "Details",
+  "dialog": {
+    "upNext": {
+      "nextEpisodePlayingIn": "Next Episode Playing in",
```

Comment:
I recommend "Next episode plays in"

Reply:
Good point, I think I can use `getCurrentItem.Type` to change the text depending on media type